src/DateTimeField.js

```diff
@@ -109,10 +109,10 @@ Ext.define('Ext.ux.DateTimeField', {
             timePicker;
 
         me.callParent();
-        timePicker = me.picker && me.picker.timePicker;
+        timePicker = me.picker;
```

Comment:
This doesn't seem like the correct fix, timePicker has to be the timePicker attached to the picker when you expand the dateField. Something else is collapsing the field, can you tell me what version of ExtJS (major and minor like 5.1.2) has this issue?